src/Ladybug_Shadow Study.py

```diff
@@ -437,10 +437,14 @@ def isShadedByFaces(self, surface):
         shade = []
         
         for item in shadowCurves:
-            shadow.append(rs.AddPlanarMesh(item))
+            meshingPar = rc.Geometry.MeshingParameters.Coarse
```

Comment:
@devngc thank you for the update. Why are you creating meshingPar each time? You can just create it once outside the loop and use it to generate the meshes. I can merge it as it is but if you can fix this last issue then that would be the better option?